src/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -814,6 +817,11 @@ public void addBase(final int offset) {
             }
         }
 
+        /** Increments the # of reads mapping to this target. */
+        public void addReadCount() {
```

Comment:
The reason was there was an `addBase` and so I wanted to make it easy to search for `add*`.  Updating to `incrementReadCount`.